docs/commands.md

```diff
@@ -0,0 +1,86 @@
+<!--[metadata]>
++++
+title = "Understand the typical commands"
+description = "Description of key notary commands"
+keywords = ["docker, notary, notary-client"]
++++
+<![end-metadata]-->
+
+# Understand the typical notary commands
+
+## Initialize a collection
+`notary init example.com/image`
```

Comment:
I think the use of `example.com` makes these a bit confusing. Could we pick a GUN that doesn't have a domain in it?